pages/start/scaffold.mdx

```diff
@@ -44,26 +44,30 @@ Open [http://localhost:5173/](http://localhost:5173/) in two tabs to see it in a
 When you are happy with your app, publish the Reflect server to [reflect.net](https://reflect.net/):
 
 ```bash copy
-npx reflect publish
+npx reflect publish --app my-app
 ```
 
-This writes a local `.env` file containing the URL of the live server, so once it's done you can run against prodution with just:
+This will output the URL for the published app. Set the environment variable for REFLECT_URL, and you can run against prodution with:
 
 ```bash copy
-npx vite
+VITE_REFLECT_URL=https://your-app-name.reflect-server.net npx vite
```

Comment:
Nit: The variable is referenced as `REFLECT_URL` above and `VITE_REFLECT_URL` here.